powershell/module/module-class.ts

```diff
@@ -105,6 +105,7 @@ export class ModuleClass extends Class {
 
         yield $this.fPipeline.assignPrivate(ClientRuntime.HttpPipeline.new(ClientRuntime.HttpClientFactory.new(System.Net.Http.HttpClient.new())));
         yield $this.fPipelineWithProxy.assignPrivate(ClientRuntime.HttpPipeline.new(ClientRuntime.HttpClientFactory.new(System.Net.Http.HttpClient.new($this.fHandler))));
+        yield "Init();";
```

Comment:
This change will have impact on both Azure and non-Azure modules. Would you call it explicitly instead of calling it in the constructor?

Reply:
Yes we are comfortable doing that.